praw/models/user.py

```diff
@@ -168,6 +199,13 @@ def subreddits(
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        To print a list of the subreddits that you are subscribed to, try:
```

Comment:
Same as above.

Reply:
I could do that. The examples I added for `contributor_subreddits` and `subreddits` show that you need to use `limit=None` to guarantee that you get more than 100 subreddits, which is related to the problem pointed out above.

Reply:
Ah I see. Could you separate it out to a different commit then and I'll merge it in in this PR?

Reply:
Sure.

I found that quarantined subreddits, which (I think) can't be accessed unless you opt-in and are logged in, show up in both modes, so I modified the wording of the note on `Redditor.moderated` a bit.